app/assets/javascripts/nilavu/controllers/topic.js.es6

```diff
@@ -185,7 +186,7 @@ export default Ember.Controller.extend(BufferedContent, {
             var self = this;
             this.set('spinnerStartIn', true);
             Nilavu.ajax('/t/' + this.get('model').id + "/start", {
```

Comment:
@vinomca-megam  As discussesd, make sure we have just one method all the actions call.

- merge getData and getDeleteData into one method.